drivers/gles3/rasterizer_canvas_gles3.cpp

```diff
@@ -1294,8 +1364,6 @@ _FORCE_INLINE_ static uint32_t _indices_to_primitives(RS::PrimitiveType p_primit
 }
 
 void RasterizerCanvasGLES3::_render_batch(Light *p_lights, uint32_t p_index, RenderingMethod::RenderInfo *r_render_info) {
-	ERR_FAIL_NULL(state.canvas_instance_batches[state.current_batch_index].command);
```

Comment:
Removing it to allow `.command_type = Item::Command::TYPE_RECT` with `.command = nullptr` used for debug redraw, added null checks for cases where `.command` is actually dereferenced.

Also seems the check here was faulty anyway, as `state.canvas_instance_batches[p_index]` should be checked instead of `state.canvas_instance_batches[state.current_batch_index]`.